packages/apollo-angular/src/types.ts

```diff
@@ -69,7 +69,7 @@ export interface MutationOptions<TData = any, TVariables = EmptyObject>
   useMutationLoading?: boolean;
 }
 
-export interface SubscriptionResult<TData> extends ExecutionResult {
+export interface SubscriptionResult<TData> extends ExecutionResult<TData> {
   data?: TData;
```

Comment:
This line `data?: TData;` can be removed with the addition of `ExecutionResult<TData>`.

Reply:
In this case data type would be `data?: TData | null; `. Is it fine to have an additional `null` there ? I believe it was the reason to have data type overridde